generator/api/src/main/java/io/fabric8/maven/generator/api/FromSelector.java

```diff
@@ -81,6 +85,40 @@ public boolean isRedHat() {
         return REDHAT_VERSION_PATTERN.matcher(version).matches();
     }
 
+    public boolean isJava11() {
```

Comment:
Won't `System.getProperty("java.version")` suffice to give you jdk version?

Reply:
What if the user has java11 on their Dev environment/machine but needs Java 8 on prod ?

Reply:
Could you please elaborate your point? I'm not sure what you mean, properties are read from jvm and would always give correct output regardless of dev/prod environment.